### DISCUSSION TITLE
[RFC Call] Agenda - January 4, 2022

### DISCUSSION THREAD

**Discussion by FredKSchott** [withastro/roadmap#62] [Announcements]
# Agenda

*

--- 

_If there is something that you would like to see discussed, please comment below. We will prioritize pending RFC, but if we have time to spare we will open the call up to get attention on and discuss [other threads and feature requests.](https://github.com/withastro/rfcs/discussions)_
If there is something else you would like to discuss, please mention it here.

**Comment by jonathantneal** [comment]
Discussed https://github.com/withastro/rfcs/pull/59

- Proposal goal is to improve source-relative URL support in Astro files.
- Primary design (`local:` url schema) is rejected, but alternative design (`local:` attribute prefix) is accepted.
- Confirmed design details:
  - `local:` attribute prefix treats the entire attribute value as the URL.
  - `local:` attribute prefix works on any attribute.
    - limiting is considered, but rejected due to the added complexity it requires (to detect the attribute name, the current element).
  - `?url` is presumed to be appended to the value.
  - attribute values with partial sources (`srcset`, `style`, some data attributes) are not handled, but could be handled in a separate, future RFC.
- Proposal is to be updated with these details before being merged.
